Directory.Build.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <Title>FsAutoComplete</Title>
     <Product>FsAutoComplete</Product>
+    <ArtifactsPath>$(MSBuildThisFileDirectory)artifacts</ArtifactsPath>
```

Comment:
I think if we go down this path, we should move to requiring .NET 8 and use the artifacts layout to simplify out outputs.